faunadb-scala/src/test/scala/faunadb/ClientSpec.scala

```diff
@@ -26,7 +31,12 @@ class ClientSpec extends FlatSpec with Matchers with BeforeAndAfterAll {
 
   val rootClient = FaunaClient(endpoint = config("root_url"), secret = config("root_token"))
 
-  val testDbName = "faunadb-scala-test"
+  val testDbName = {
```

Comment:
We were using the same database name for every run before (thus we were dropping it as well in the `beforeAll` method before for the suite started). I've included a suffix for each run now in case we want to take a look a the database in case of error. The database is dropped as always when all of the test cases in the suite are finished.